.github/workflows/build-l10n.yml

```diff
@@ -0,0 +1,34 @@
+name: Build
+
+on:
+  push:
+    branches:
+      - l10n_master
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    name: Compile updated translations and commit changes
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python 3.8
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+
+      - name: Install dependencies
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y gettext
+          python -m pip install --upgrade pip
+          pip install psycopg2==2.8.6
+          pip install .
```

Comment:
install from the requirement.txt file instead? `psycopg-binary` is included there

Reply:
Noted